.buildkite/pipeline.provision.yml

```diff
@@ -14,6 +14,10 @@ steps:
           command: preview
           project_dir: pulumi/grapl
           stack: grapl/testing
+      - grapl-security/pulumi#v0.1.2:
+          command: preview
+          project_dir: pulumi/integration_tests
+          stack: grapl/integration-tests/testing
```

Comment:
We should either use the 3-part fully-qualified stack identifier everywhere, or change this to `grapl/testing`. Specifying the `project_dir` does the job of setting the project. Having 2- and 3-part identifiers mixed together is likely to be confusing to new people ("Is this difference significant? _Have I been doing it wrong the whole time?_").

(I suspect this is probably only an issue in this repository, where we have a project that is the same name as our Pulumi organization, leading to some first-glance confusion: "Wait, I thought we were in the `integration-tests` project; why does it say `grapl`?... _oooooh right_, that's the _organization_ name.")